### PR TITLE
Use where.exe on Windows instead of which

### PR DESCRIPTION
**Pull request by gmittert**
Windows (without msys) doesn't have `where`, instead, use the native `which`
to locate tools.

### CODE REVIEW

**Comment by gmittert** [comment]
Rebased!

@swift-ci please test

File: Sources/ISDBTibs/TibsToolchain.swift [hunk 1]
```diff
@@ -160,15 +167,28 @@ extension TibsToolchain {
 public func findTool(name: String) -> URL? {
 #if os(macOS)
   let cmd = ["/usr/bin/xcrun", "--find", name]
+#elseif os(Windows)
+  var buf = [WCHAR](repeating: 0, count: Int(MAX_PATH))
+  GetWindowsDirectoryW(&buf, DWORD(MAX_PATH))
+  let foo = DWORD(0)
```
Comment (gmittert) [inline Sources/ISDBTibs/TibsToolchain.swift]: Blah.. this needs to be removed..

Comment (gmittert) [reply Sources/ISDBTibs/TibsToolchain.swift]: Removed

**Comment by gmittert** [comment]
@swift-ci please test

**Comment by gmittert** [comment]
@swift-ci please test

File: Sources/ISDBTibs/TibsToolchain.swift [hunk 2]
```diff
@@ -201,8 +217,13 @@ public func findTool(name: String) -> URL? {
   guard var path = String(data: data, encoding: .utf8) else {
     return nil
   }
+#if os(Windows)
+  let paths = path.split { $0.isNewline }
+  path = String(paths[0]).trimmingCharacters(in: .whitespacesAndNewlines)
```
[the remark this answers is not in the document]
Comment (gmittert) [reply Sources/ISDBTibs/TibsToolchain.swift]: Seems reasonable :) Changed!

**Comment by benlangmuir** [comment]
@swift-ci please test

**Comment by benlangmuir** [comment]
@swift-ci please test

**Comment by gmittert** [comment]
@benlangmuir By the way, thanks for being so quick to get to all of these 👍

**Pull request merged**